templates/index.html

```
@@ -0,0 +1,89 @@
+                <li>{{ link }}</li>
+            {% endfor %}
+        </ul>
+        {% else %}
+        <p>No related notes found.</p>
+        {% endif %}
+    </div>
+
+    <!-- FOOTER -->
+    <div class="footer">
+        <p>Powered by AOSSIE</p>
+        <img src="{{ url_for('static', filename='images/aossie-logo.svg') }}" class="aossie-logo">
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)